src/crimson/osd/object_context.h

```diff
@@ -222,4 +227,4 @@ class ObjectContextRegistry : public md_config_obs_t  {
                           const std::set <std::string> &changed) final;
 };
 
-}
+} // namespace crimson::osd
```

Comment:
I actually find these annotations to be quite unhelpful.

Reply:
Note - following our meeting: I've verified that clang-format indeed fixes wrong //namespace end notes.
And for me - they are helpful.